application/src/main/webapp/vue-app/components/ExoChatContact.vue

```diff
@@ -117,7 +117,8 @@ export default {
         donotdisturb: this.$t('exoplatform.chat.donotdisturb'),
         invisible: this.$t('exoplatform.chat.invisible'),
         offline: this.$t('exoplatform.chat.button.offline')
-      }
+      },
+      spaceGroupUri: null,
```

Comment:
No need to add this as data

Reply:
it is a global variable when it is putted into the method its value never changed